MISC.rst

```diff
@@ -0,0 +1,63 @@
+=============
```

Comment:
have questions about a 'new' .rst file.     I know the README.rst is already long... but ... unless we have an overall plan, I don't want to start breaking them up or having additional files... it will make that information effectively 'hidden'.

So, consider placing a new section in the existing README for now.   Maybe breaking up the docs is a separate issue/effort.

Reply:
Should it be treated as part of the natural flow of the README?  Most of the existing topics are tutorials and guides for usage, and this is more a guide for setup and test of S3/Parallel Transfer.

Reply:
well, exactly... i think there may be a good cause for breaking things into a book or chapters or pages...   but i'd like to understand how pypi deals with that... can they be linked back together, etc...   before we do it.

yes just add a section at the bottom of the main README please